freeipa/src/main/resources/freeipa-salt/salt/freeipa/templates/freeipa_backup.conf.j2

```diff
@@ -2,5 +2,6 @@
 backup_location={{freeipa.backup_location}}
 backup_platform={{freeipa.backup_platform}}
 azure_instance_msi={{freeipa.azure_instance_msi}}
+gcp_service_account={{freeipa.gcp_service_account}}
```

Comment:
is this a valid option for the backup, or just pre-filled and will be used somehow later?

Reply:
I guess so because the azure instance msi is also presented here